includes/admin/class-sensei-extensions.php

```diff
@@ -168,15 +168,29 @@ private function add_installed_extensions_properties( $extensions ) {
 			$wccom_subscriptions = WC_Helper::get_subscriptions();
 		}
 
+		$aliases = [
+			'sensei-pro/sensei-pro.php' => [
+				'woothemes-sensei/woothemes-sensei.php',
+			],
+		];
+
 		// Includes installed version, whether it has update and WC.com metadata.
 		$extensions = array_map(
-			function( $extension ) use ( $installed_plugins, $wccom_subscriptions ) {
-				$extension->is_installed = isset( $installed_plugins[ $extension->plugin_file ] );
-				$extension->is_activated = $extension->is_installed && is_plugin_active( $extension->plugin_file );
+			function( $extension ) use ( $installed_plugins, $wccom_subscriptions, $aliases ) {
+				$plugin_file = $extension->plugin_file;
+
+				$plugin_file_aliases   = array_key_exists( $plugin_file, $aliases ) ? $aliases[ $plugin_file ] : [];
+				$plugin_file_aliases[] = $plugin_file;
```

Comment:
The `$plugin_file` (which is equal to `$extension->plugin_file` is always added at the end of the check because we need to check for aliases first, so if an alias is found, we just stop the processing without checking for the rest.